### ISSUE TITLE
Bug: Session quickly and repeatedly expires on Personal Desktop for MacOS

### ISSUE THREAD

**Issue by mattouille** [mattermost/focalboard#2156]
## Steps to reproduce the behavior

1. Open Focalboard
2. Do any action or wait for up to two minutes
3. Session expires

## Expected behavior

On desktop the session should never expire

## Screenshots

![Screen Shot 2022-01-21 at 11 32 53 AM](https://user-images.githubusercontent.com/8207468/150589438-139ac094-16c3-43bc-914e-1127fdb9400c.png)

## Edition and Platform

 - Edition: Personal Desktop
 - Version: v0.12.0
 - Browser and OS: MacOS Big Sur 11.2.3

## Additional context

N/A

**Comment by chenilim** [comment]
Does this repro for you @ogi-m?

**Comment by ogi-m** [comment]
@jgilliam17 tested on macOS and couldn't repro, the session doesn't expire.
cc @chenilim

**Comment by chenilim** [comment]
Thanks @ogi-m. @mattouille - does this consistently repro if you quit and reopen the app? There is a session key, but (in theory) that should last for 30 days.

**Comment by mattouille** [comment]
> Thanks @ogi-m. @mattouille - does this consistently repro if you quit and reopen the app? There is a session key, but (in theory) that should last for 30 days.

Yeah, it seems to persist even after a system restart. On my Linux system, however, this doesn't occur at all.

**Comment by wuwinson** [comment]
Thanks @mattouille! Any additional details or observations will help us determine the root cause. If anyone else has similar cases, please feel free to add to this ticket.

**Comment by mattouille** [comment]
Is there anywhere that Focalboard logs? I was looking in some common destinations and didn't see any.
